engine/apps/public_api/tests/test_incidents.py

```diff
@@ -38,6 +38,7 @@ def construct_expected_response_from_incidents(incidents):
                 "resolved_at": resolved_at,
                 "acknowledged_at": acknowledged_at,
                 "title": None,
+                "permalink": None,
```

Comment:
wondering if it is worthwhile adding a test case here to make sure that this is equal to the slack message URL when the alert group's `slack_message` is not `None`?

Reply:
Yeah I guess we can add a test on this, not so critical though

Reply:
Nice idea to add links for multiple messengers @Matvey-Kuk!

Reply:
good suggestion @Matvey-Kuk! will refactor to take this into consideration

Reply:
Renamed the field in the public API `AlertGroup` serializer from `permalink` to a dict named `permalinks`.

*Note*: I think it eventually would make sense to rename the `permalink` property on the `AlertGroup` model (which right now represents the slack message permalink). I avoided doing that in this PR to avoid making a bigger/out-of-scope PR and instead just added a TODO note in the relevant code. Thoughts?

Reply:
@joeyorlando migrating AlertGroup model is dangerous because it's HUGE and may cause long downtime while migrating, but it's good not to collect tech debt. So I'm ok with any decision you make :)